pkg/reconciler/mtnamespace/controller.go

```diff
@@ -46,19 +51,40 @@ func NewController(
 	cmw configmap.Watcher,
 ) *controller.Impl {
 
+	logger := logging.FromContext(ctx)
 	namespaceInformer := namespace.Get(ctx)
 	brokerInformer := broker.Get(ctx)
 
+	recorder := controller.GetEventRecorder(ctx)
+	if recorder == nil {
```

Comment:
same comment as https://github.com/knative/eventing/pull/2819

Reply:
I am doing the thinest change.